server-embedded/src/main/java/org/exoplatform/chat/services/mongodb/UserMongoDataStorage.java

```
@@ -609,14 +615,25 @@ private SpaceBean getSpace(String roomId, String dbName)
       spaceBean.setRoom(roomId);
       spaceBean.setId(doc.get("space_id").toString());
       spaceBean.setDisplayName(doc.get("displayName").toString());
+
+
+
+
       spaceBean.setGroupId(doc.get("groupId").toString());
       spaceBean.setShortName(doc.get("shortName").toString());
+      //spaceBean.setAdmin((doc.get("user").toString()));
       if (doc.containsField("timestamp"))
       {
         spaceBean.setTimestamp(((Long)doc.get("timestamp")).longValue());
       }
     }
 
+    SpaceService spaceService = CommonsUtils.getService(SpaceService.class);
+    try {
+      Space space1 = spaceService.getSpaceByDisplayName(spaceBean.getDisplayName());
+      spaceBean.setAdmins(space1.getManagers());
+    }catch (Exception e){}
```

No empty catch please. Should log an error.